espnet/tts/pytorch_backend/tts.py

```diff
@@ -316,6 +316,7 @@ def train(args):
         use_second_target=args.use_second_target,
         preprocess_conf=args.preprocess_conf,
         preprocess_args={'train': True}  # Switch the mode of preprocessing
```

Comment:
```suggestion
        preprocess_args={'train': True},  # Switch the mode of preprocessing
```

Reply:
Ah, this is due to cherry-picking conflict. Will push a fix shortly.